src/prioritized_cache.cc

```diff
@@ -0,0 +1,23 @@
+#include "titan/prioritized_cache.h"
+
+namespace rocksdb {
+namespace titandb {
+
+PrioritizedCache::PrioritizedCache(std::shared_ptr<Cache> cache,
+                                   Cache::Priority priority)
+    : cache_(cache), priority_(priority) {}
+
+PrioritizedCache::~PrioritizedCache() {}
+
+std::shared_ptr<Cache> PrioritizedCache::GetCache() { return cache_; }
+
+Status PrioritizedCache::Insert(const Slice& key, void* value, size_t charge,
+                                void (*deleter)(const Slice& key, void* value),
+                                Cache::Handle** handle,
+                                Cache::Priority /*priority*/) {
+  if (!cache_) return Status::NotFound();
```

Comment:
let's assume `cache_` always present.

Reply:
ok, so I add an assert(cache_) here to replace check?